tests/test_jsdoc_analysis/test_jsdoc.py

```diff
@@ -86,8 +86,16 @@ def test_class(self):
         assert cls.path == Pathname(['./', 'class.', 'Foo'])
         assert cls.filename == 'class.js'
         assert cls.description == 'This is a long description that should not be unwrapped. Once day, I was\nwalking down the street, and a large, green, polka-dotted grand piano fell\nfrom the 23rd floor of an apartment building.'
-        assert cls.line == 15  # Not ideal, as it refers to the constructor, but we'll allow it
-        assert cls.examples == ['Example in constructor']  # We ignore examples and other fields from the class doclet so far. This could change someday.
+        # Not ideal, as it refers to the constructor, but we'll allow it
+        assert cls.line in (
+            8,  # jsdoc 4.0.0
```

Comment:
The changes in the test values suggests that maybe this fixes the issue in the comment where with jsdoc 3.6.3, we were getting data about the constructor and now in jsdoc 4.0.0, we're getting data about the class.